### ISSUE TITLE
Using generator and discriminator loss to evaluate GANs.

### ISSUE THREAD

**Issue by Sinha-Raunak** [soumith/ganhacks#43]
Hey!

I was studying upon evaluation metric for GANs.
Was a little confused as to why the loss of the generator and discriminator are not good measures to evaluate the performance of GANs. 

Thanks

**Comment by LKSPNM** [comment]
The generator and discriminator loss doesn't necessary represents the image convergence results. At least not the binary cross-entropy loss function. The WGAN proposes a EM evaluation as the loss function, this one converge and can represent very well the convergence of the network result. If you don`t want to implement the WGAN loss try using the MSE error for evaluate the images or SSIM(Structural Simiarity Index) for image evaluation.
